nix/rust-package.nix

```diff
@@ -84,7 +84,7 @@ let
   sharedArgs =
     if runTests then sharedArgsBase // {
       # exclude hopr-socks-server because it requires access to the internet
-      cargoTestExtraArgs = "--workspace --exclude hopr-socks-server";
+      cargoTestExtraArgs = "--workspace -F runtime-async-std -F runtime-tokio --exclude hopr-socks-server";
```

Comment:
This should not be passed for the `release` build though.

Reply:
It's only under `if runTests` if I understand correctly